### DISCUSSION TITLE
Paddings

### DISCUSSION THREAD

**Discussion by tbmuc** [ImageMagick/ImageMagick#2088] [Help]
Hello, 
I would like automatically create an JPG, which has the following  structure:

Padding to the top
Text1
Padding
Image
Padding
Text2
Padding to the bottom

I'm not sure how to create the paddings.

**Comment by snibgo** [comment]
What version of IM, on what platform?

You can create seven images, and append them vertically. For example, with just three images:
`magick -size 100x50 label:Hello -size 1x34 xc: -size 100x50 label:World -append x.jpg`

**Comment by tbmuc** [comment]
Thank you very much. IM 7 Windows 10 64x

I'm fighting, could you take a look at this: 

convert -size 822x400  %1.png -trim +repage -resize 750x750^ -background white -alpha remove -alpha off -background white -gravity south -extent 822x850  -size 822x200 -stroke black -font "C:\Program Files\ImageMagick-7.0.10-Q16\Hey August.ttf" -pointsize 100  white -size 822x200 -label %2  -undercolor white -append z.jpg

It should bring a png image into the format 750x750, extend it to 822x850 and add a 100 px text into a 822x200 box below.

However, the result is the image without the text

**Reply by GeeMack** [reply]
It looks like you're using "-label" incorrectly. Try making your text label like this ` label:"%2" `
